props/saber_BC_buttons.h

```diff
@@ -364,14 +1095,31 @@ Turn OFF without postoff - Turn OFF while pointing up.
 #define THRUST_GESTURE
 #endif
 
+#ifndef BC_MAIN_BLADE
+#define BC_MAIN_BLADE 1
+#endif
+
+#ifndef BC_SECOND_BLADE
+#define BC_SECOND_BLADE 2
+#endif
+
+#if BC_MAIN_BLADE == BC_SECOND_BLADE
```

Comment:
Only check this if BC_DUAL_BLADES is defined

Reply:
ok